.vscode/settings.json

```diff
@@ -0,0 +1,7 @@
+{
+    "files.associations": {
+        "*.out": "yaml",
+        "pmem.h": "c",
+        "psemaphore.h": "c"
+    }
+}
```

Comment:
The same here, this file needs to be removed.

Reply:
okay, i delete it